tests/test_mapdl.py

```diff
@@ -178,7 +178,7 @@ def test_server_version(mapdl):
 def test_global_mute(mapdl):
     mapdl.mute = True
     assert mapdl.mute is True
-    assert mapdl.prep7() == ""
+    assert mapdl.prep7() == None
```

Comment:
```suggestion
    assert mapdl.prep7() is None
```
source: https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none

Reply:
Accepted. Thank you.